js/bitfinex.js

```diff
@@ -379,10 +379,8 @@ module.exports = class bitfinex extends Exchange {
                     'spot': 'exchange',
                     'margin': 'trading',
                     'funding': 'deposit',
-                    'exchange': 'exchange',
-                    'trading': 'trading',
-                    'deposit': 'deposit',
-                    'derivatives': 'trading',
+                    'future': 'exchange',
+                    'swap': 'trading',
```

Comment:
Of all derivatives-types, I think bitfinex has just the `swap` type (perpetual futures aka perpetual swaps) and no `future` type (no expiring future contracts)